chat/chat.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Transliteration</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css">
+
+    <link rel="stylesheet" href="./chat.css">
+    <script defer src="./chat.js"></script>
+</head>
+<body class="container bg-primary">
+    <div class="row justify-content-center align-items-center">
+        <div class="col-lg-2 text-white">
+            <label for="second-input">User 2:</label>
+            <textarea class="form-control resize-none" name="second-input" id="second-input" placeholder="Message text"></textarea>
+            <button onclick="addSecondUserMessage()">Send</button>
```

Comment:
Better use addEventListener in Js, instead of onclick usage in HTML

Template should avoid any specific logic (call of JS function is actually logic, that better be stored in one place - JS file)